ansible/roles/graylog-mongodb-import/defaults/main.yml

```diff
@@ -1,29 +1,4 @@
 graylog_mongo_db: "graylog"
 mongo_master: "{{ groups['graylog'][0] }}"
 mongo_json_data: /tmp/mongo_json_data
-graylog_mongo_collections_drop: true
-graylog_mongo_collections:
-  - cluster_config
-  - content_packs
-  - content_packs_installations
-  - event_definitions
-  - event_notifications
-  - event_notification_status
-  - grants
-  - grok_patterns
-  - index_sets
-  - inputs
-  - lut_caches
-  - lut_data_adapters
-  - lut_tables
-  - pipeline_processor_pipelines
-  - pipeline_processor_pipelines_streams
-  - pipeline_processor_rules
-  - roles
-  - scheduler_job_definitions
-  - scheduler_triggers
-  - searches
-  - streamrules
-  - streams
-  - users
-  - views
+graylog_mongo_collections: "cluster_config, content_packs, content_packs_installations, event_definitions, event_notifications, event_notification_status, grants, grok_patterns, index_sets, inputs, lut_caches, lut_data_adapters, lut_tables, pipeline_processor_pipelines, pipeline_processor_pipelines_streams, pipeline_processor_rules, roles, scheduler_job_definitions, scheduler_triggers, searches, streamrules, streams, users, views"
```

Comment:
Please keep it as an ansible list. It'll be more readable, and idiomatic.
Else it'll become like our yarn provision ansible role.

Reply:
Sure we will change in next iteration as it needs changes acorss multiple files.